test/JDBC/expected/BABEL-937.out

```diff
@@ -0,0 +1,210 @@
+-- Not Append
+--NOT NULL, LAX, KP  
+SELECT JSON_MODIFY('{"name":"John","skills":["C#","SQL"]}','$.name','Mike');
```

Comment:
Please add some examples that use fully formatted JSON (such as what you'd get with `jsonb_pretty()`).

Reply:
Added the test cases.